src/Autowhatever.js

```diff
@@ -98,19 +99,22 @@ export default class Autowhatever extends Component {
       const onClickFn = onClick ?
         event => onClick(event, { sectionIndex, itemIndex }) :
         noop;
+      const isFocused = sectionIndex === focusedSectionIndex && itemIndex === focusedItemIndex;
       const itemProps = {
         id: this.getItemId(sectionIndex, itemIndex),
         role: 'option',
-        ...theme(itemIndex, 'item', sectionIndex === focusedSectionIndex &&
-                                    itemIndex === focusedItemIndex &&
-                                    'itemFocused'),
+        ...theme(itemIndex, 'item', isFocused && 'itemFocused'),
         ...itemPropsObj,
         onMouseEnter: onMouseEnterFn,
         onMouseLeave: onMouseLeaveFn,
         onMouseDown: onMouseDownFn,
         onClick: onClickFn
       };
 
+      if (isFocused) {
+        itemProps.ref = 'focusedItem';
+      }
```

Comment:
I'd like to avoid mutation if possible (especially, because `itemProps` is `const`). Would this work:
```
const itemProps = {
  ref: isFocused ? 'focusedItem' : null,
  ...
};
```
?